packages/core/src/Stores/traders-hub-store.js

```diff
@@ -382,6 +390,13 @@ export default class TradersHubStore extends BaseStore {
             return;
         }
 
+        if (this.financial_restricted_countries) {
+            this.available_mt5_accounts = this.available_cfd_accounts.filter(
+                account => account.market_type === 'financial'
```

Comment:
```suggestion
                account => account.market_type === 'financial' && account.platform === CFD_PLATFORMS.MT5
```

Reply:
What is the purpose of including the line. I've tried including and excluding it before but it doesn't make any difference.

Reply:
Extra check like the below condition